packages/expo-updates/ios/EXUpdates/AppLoader/AppLoaderTask.swift

```diff
@@ -497,3 +540,7 @@ public final class AppLoaderTask: NSObject {
     }
   }
 }
+
+// swiftlint:enable closure_body_length
+// swiftlint:enable force_unwrapping
+// swiftlint:enable superfluous_else
```

Comment:
no need to re-renable swiftlint at the end of files. ditto for other files too

Reply:
actually SwiftLint now complains if you do not do this.